docs/pages/attributesJSON.rst

```diff
@@ -46,9 +46,125 @@ Below are the supported JSON tags for Physics Manager Attributes templates, and
+    :code: json
+
+'Scene Instance Parameters'
+---------------------------
+
+"translation_origin"
+  - string
+  - Specifies scene instance default for translation origin. Values are case-insensitive. Supports "asset_local", where the origin is defined in the individual asset's local frame, "com", where the asset's com is the origin for transformations, or "unknown" if the value is unknown. This value can be overridden for the stage and any object instances in the scene.
+
+"default_lighting"
+  - string
+  - The handle of the lighting setup configuration, defined in the Scene Dataset Configuration, that should be usedd when instantiating lights in the scene.
+
+"navmesh_instance"
+  - string
+  - The alias of the scene's navmesh asset, which is mapped to a file handle in the Scene Dataset Configuration.
+
+"semantic_scene_instance"
+  - string
+  - The alias of the scene's semantic scene descriptor file, which is mapped to a file handle in the Scene Dataset Configuration.
+
+'Scene Instance Stage and Object parameters'
+--------------------------------------------
+
+The Scene Instance Attributes specifies all the required parameters to instantiate the scene's stage and objects, such as source attributes handle and intial transformation. These creation parameters are called Scene Object Instance Attributes, and they are tagged in the Scene Instance Attributes configuration file as follows:
+
+"stage_instance"
+  - json object
+  - the required Scene Object Instance parameters to create the stage in the scene.
+
+"object_instances"
+  - array of json ojbects
+  - each element of the array is a Scene Object Instance parameters required to create a single object in the scene.
+
+The JSON tags the Scene Object Instance Attributes support are as follows:
+
+"template_name"
+  - string
+  - Dataset-unique name of the template for the stage or object attributes desired to instantiate. Since attributes templates can encode full file paths, their handles can be very long; therefore, the "template_name" field supports substring lookup, and an incomplete handle can be used for this field so long as it uniquely maps to the full handle specified in the Scene Dataset Attributes.
+
+"translation_origin"
+  - string
+  - Specifies the translation origin to use for the instantiation of the specified object. Values are case-insensitive. Supports "asset_local", where the origin is defined in the individual asset's local frame, "com", where the asset's com is the origin for transformations, or "unknown" if the value is unknown. If this value is specified, and not set to "unknown", it will override the top-level default, if any exists. Currently stages do not support instantiation transformations.
+
+"motion_type"
+  - string
+  - The desired motion type of the object being instanced. Values are case-insensitive. Values supported are "static", "kinematic", and "dynamic". This field is ignored for stages, which are always created with STATIC motion type.
```

Comment:
Link to somewhere where we describe what "static", "kinematic", and "dynamic" mean.